include/experimental/__p1673_bits/blas1_vector_norm2.hpp

```diff
@@ -56,37 +57,19 @@ template<class in_vector_t,
 void vector_norm2(in_vector_t x,
                   Scalar& result)
```

Comment:
The proposal has us taking `T init` and returning `T`, just as `std::reduce` does.  We no longer take the result as as output argument.  I'll file a separate issue for that.

Reply:
The separate issue: https://github.com/kokkos/stdBLAS/issues/52